rewrite-maven/src/main/java/org/openrewrite/maven/cache/RocksdbMavenPomCache.java

```diff
@@ -103,22 +104,23 @@ static synchronized RocksCache getCache(String workspace) {
     CacheResult<MavenMetadata> UNAVAILABLE_METADATA = new CacheResult<>(CacheResult.State.Unavailable, null);
     CacheResult<MavenRepository> UNAVAILABLE_REPOSITORY = new CacheResult<>(CacheResult.State.Unavailable, null);
 
-    public RocksdbMavenPomCache(@Nullable File workspace) {
+    public RocksdbMavenPomCache(@Nullable String folder) {
 
-        assert workspace != null;
+        assert folder != null;
 
-        if(!workspace.exists() && !workspace.mkdirs()) {
-            throw new IllegalStateException("Unable to find or create maven pom cache at " + workspace);
-        } else if (!workspace.isDirectory()) {
+        File relativePath = new File(System.getProperty("user.home") + "/.rewrite/cache/" + folder);
```

Comment:
It's not appropriate for `RocksdbMavenPomCache` to insist on locating itself inside the user's home directory. The choice of where the cache goes should not be taken away from the caller.

Reply:
Fair, appended `/.pom` to the path instead.